docs/source/model-registry.rst

```diff
@@ -600,6 +605,72 @@ save, log, register, and load from the Model Registry and score.
     <Yay!! Another good phone interview. I nailed it!!> -- {'neg': 0.0, 'neu': 0.446, 'pos': 0.554, 'compound': 0.816}
     <This is INSANE! I can't believe it. How could you do such a horrible thing?> -- {'neg': 0.357, 'neu': 0.643, 'pos': 0.0, 'compound': -0.8034}
 
+Using Registered Model Aliases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note::
+    Registered model aliases are experimental and subject to change.
+
+Model aliases in the MLflow Model Registry allow you to assign a mutable, named reference to a particular version within a
+registered model. You can use aliases to specify which model versions are deployed in a given environment in your model
+training workflows (e.g. specify the current ``Champion`` model version that should serve the majority of production traffic),
+and then write inference workloads that target that alias ("make predictions using the ``Champion`` version"). The example
+workflow below captures this idea.
+
+
+**Mark model for deployment using aliases**
+
+We can set a ``Champion`` alias to point to a newly trained model version, indicating to downstream inference workloads that they
+should use this model version to make predictions on production traffic:
+
+.. code-block:: python
+
+    # Register a new model and version
+    from random import random, randint
+    from sklearn.ensemble import RandomForestRegressor
+
+    import mlflow
+    import mlflow.sklearn
+
+    with mlflow.start_run(run_name="YOUR_RUN_NAME") as run:
+        params = {"n_estimators": 5, "random_state": 42}
+        sk_learn_rfr = RandomForestRegressor(**params)
+
+        # Log parameters and metrics using the MLflow APIs
```

Comment:
I'd recommend removing some of the extraneous `log_params` etc to keep this doc section focused on aliases, especially given the length of the doc page already